src/js/actions/type.js

```diff
@@ -114,6 +136,32 @@ define(function (require, exports) {
     };
     setPostScript.reads = [locks.JS_APP, locks.JS_TOOL];
     setPostScript.writes = [locks.PS_DOC, locks.JS_DOC, locks.PS_APP, locks.JS_POLICY];
+    setPostScript.modal = true;
+
+    /**
+     * Update the type face (in terms of a type family and type style) of the given
+     * layers in the given document. 
+     *
+     * @param {Document} document
+     * @param {Immutable.Iterable.<Layers>} layers
+     * @param {string} family The type face family name, e.g., "Helvetica Neue"
+     * @param {string} style The type face style name, e.g., "Oblique"
+     * @return {Promise}
+     */
+    var updateFace = function (document, layers, family, style) {
```

Comment:
Ah I see.. 

I think all the set actions in here should transfer to the updates then, instead of calling them. This is gonna be one busy action queue otherwise.